### PR TITLE
add README file

### PR DESCRIPTION
**Pull request by pyMiron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Python playground notes
+
+TBD
```
Comment (gromver) [inline README.md:3]: What is TBD about?

Comment (pyMiron) [reply README.md:3]: its means to be descripted

Comment (gromver) [reply README.md:3]: ok, thanks

**Review by gromver — APPROVED** [review]
everything fine

**Pull request merged**